caldav/davclient.py

```diff
@@ -539,14 +539,38 @@ def request(self, url, method="GET", body="", headers={}):
             auth_type = r.headers['WWW-Authenticate']
             auth_type = auth_type[0:auth_type.find(" ")]
 
+            error.assert_(auth_type in ('Basic', 'Digest'))
```

Comment:
I think this is moot?  There is a `raise NotImplementedError` further below that handles unsupported auth types

Reply:
Hmm, I *think* this is probably a copy&past error left over from the massive merge conflict I got when attempting to rebase my v0.8 fix on top of master.  After which I did a reimplementation from scratch.

The line should be removed.  I'll refresh my PR accordingly